client/mining_common.h

```diff
@@ -22,7 +22,7 @@ struct xdag_pool_task {
 };
 
 extern struct xdag_pool_task g_xdag_pool_task[2];
-extern uint64_t g_xdag_pool_task_index;
+extern uint64_t g_xdag_pool_task_index; // just a never initialized index, start value is not important
```

Comment:
You can just fix it. Not necessary to describe the error.

Reply:
i don't think it is an error, it is just the way it works

Reply:
This variable is always incremented. Also task index is writed to log.

Reply:
let me check

Reply:
just set default value 
g_xdag_pool_task_index = 0
No need to write such long comments about nothing.

Reply:
Take into account - global variables are always initialized with default value (0)

Reply:
didn't know, fixed.